snippets.go

```diff
@@ -111,6 +124,7 @@ type CreateSnippetOptions struct {
 	Description *string          `url:"description,omitempty" json:"description,omitempty"`
 	Content     *string          `url:"content,omitempty" json:"content,omitempty"`
 	Visibility  *VisibilityValue `url:"visibility,omitempty" json:"visibility,omitempty"`
+	Files       []*SnippetFile   `url:"files,omitempty" json:"files,omitempty"`
```

Comment:
I know this seems weird, but this needs to be `*[]*SnippetFile` to keep it consistent with the rest of the package.